packages/runtime/src/lib/constants.ts

```diff
@@ -189,7 +189,7 @@ export const TxArrFields: { [key: number]: Set<string> } = {
 TxArrFields[3] = new Set([...TxArrFields[2], "Assets", "Applications"]);
 TxArrFields[4] = cloneDeep(TxArrFields[3]);
 TxArrFields[5] = cloneDeep(TxArrFields[4]);
-TxArrFields[6] = cloneDeep(TxArrFields[5]);
+TxArrFields[6] = new Set([...TxArrFields[5], "Logs"]);
```

Comment:
In the opcode document, I see we have Logs for itxn in version 5. However you add it in teal 6. Is this still work?
https://github.com/algorand/go-algorand/blob/master/data/transactions/logic/TEAL_opcodes.md#txna-f-i

Reply:
It should be 5 i think